client/src/components/Profile/EditAvailability.tsx

```diff
@@ -0,0 +1,154 @@
+                  />
+                  <h4 className={classes.labelStyle}>Mondays </h4>
+                </div>
+                <div className={classes.checkBoxAndLabel}>
+                  <Checkbox
+                    name="Tuesday"
+                    checked={values.availability?.days?.Tuesday}
+                    onChange={(c, v) => setFieldValue('availability[days][Tuesday]', v)}
+                    inputProps={{ 'aria-label': 'primary checkbox' }}
+                  />
+                  <h4 className={classes.labelStyle}>Tuesdays </h4>
+                </div>
+                <div className={classes.checkBoxAndLabel}>
+                  <Checkbox
+                    name="Wednsday"
+                    checked={values.availability?.days?.Wednesday}
+                    onChange={(c, v) => setFieldValue('availability[days][Wednesday]', v)}
+                    inputProps={{ 'aria-label': 'primary checkbox' }}
+                  />
+                  <h4 className={classes.labelStyle}>Wednesdays </h4>
+                </div>
+
+                <div className={classes.checkBoxAndLabel}>
+                  <Checkbox
+                    name="Thursday"
+                    checked={values.availability?.days?.Thursday}
+                    onChange={(c, v) => setFieldValue('availability[days][Thursday]', v)}
+                    inputProps={{ 'aria-label': 'primary checkbox' }}
+                  />
+                  <h4 className={classes.labelStyle}>Thursdays </h4>
+                </div>
+
+                <div className={classes.checkBoxAndLabel}>
+                  <Checkbox
+                    name="Friday"
+                    checked={values.availability?.days?.Friday}
+                    onChange={(c, v) => setFieldValue('availability[days][Friday]', v)}
+                    inputProps={{ 'aria-label': 'primary checkbox' }}
+                  />
+                  <h4 className={classes.labelStyle}>Fridays </h4>
+                </div>
+
+                <div className={classes.checkBoxAndLabel}>
+                  <Checkbox
+                    name="Saturday"
+                    checked={values.availability?.days?.Saturday}
+                    onChange={(c, v) => setFieldValue('availability[days][Saturday]', v)}
+                    inputProps={{ 'aria-label': 'primary checkbox' }}
+                  />
+                  <h4 className={classes.labelStyle}>Saturdays </h4>
+                </div>
+
+                <div className={classes.checkBoxAndLabel}>
+                  <Checkbox
+                    name="Sunday"
+                    checked={values.availability?.days?.Sunday}
+                    onChange={(c, v) => setFieldValue('availability[days][Sunday]', v)}
+                    inputProps={{ 'aria-label': 'primary checkbox' }}
+                  />
+                  <h4 className={classes.labelStyle}>Sundays </h4>
+                </div>
+              </Grid>
```

Comment:
Would be nice to have this conditionally rendered via a loop of the days of the week as opposed to hard coding it, would reduce our lines of code and make this a bit more readable.